webapp/packages/core-sdk/src/queries/fragments/DatabaseDriver.gql

```diff
@@ -22,8 +22,12 @@ fragment DatabaseDriver on DriverInfo {
   applicableNetworkHandlers
   configurationTypes
 
+  mainProperties {
+    ...DriverMainPropertyInfo
+  }
```

Comment:
it's better to create a separate resource that will load this data when required (replacement for old approach with includes)